versioneer.py

```diff
@@ -0,0 +1,1857 @@
+# Version: 0.18
```

Comment:
So we can just add this as a dependency rather than copying the source here

Reply:
so versioneer.py should also be added to gitignore as well?

Reply:
We should probably also take that approach of adding `versioneer` as a dependency in `syft-proto` (which currently has the source included in the root of the repo.)

Reply:
Adding versioneer.py source to repo is how it's supposed to be installed: https://github.com/warner/python-versioneer/blob/master/INSTALL.md
Unless I misunderstood it :)
Perhaps this is to enable getting versions w/o installing any deps upfront.

Reply:
I second @vvmnnnkv, versioneer the package has only one usage `versioneer install` which adds versioneer.py.
I tried to work around it with #2949 to avoid having the file being in the repo. But now things break in regards of style.